ArcGISRuntimeSDKQt_CppSamples/Scenes/ChangeAtmosphereEffect/ChangeAtmosphereEffect.h

```diff
@@ -28,29 +28,37 @@ class SceneQuickView;
 
 #include <QObject>
 
+#include "SceneViewTypes.h"
+
 class ChangeAtmosphereEffect : public QObject
 {
   Q_OBJECT
-
+  Q_PROPERTY(AtmosphereEnum atmosphereEffect READ atmosphereEffect WRITE setAtmosphereEffect NOTIFY atmosphereEffectChanged)
   Q_PROPERTY(Esri::ArcGISRuntime::SceneQuickView* sceneView READ sceneView WRITE setSceneView NOTIFY sceneViewChanged)
-
 public:
+  enum AtmosphereEnum {
+    None = static_cast<int>(Esri::ArcGISRuntime::AtmosphereEffect::None),
```

Comment:
this is fine - but I wonder if we can actually just `#include` the Esri enum and then register/expose that?